### PR TITLE
Configure the request retry policy after calling Huawei Cloud SDK traffic restriction

### LINKED ISSUE #440: Calling HuaweiCloud SDK time limit stream error
Configure a request retry strategy, wait for 60 seconds after flow restriction before reissuing the request.

### PR DESCRIPTION
**Pull request by jinyangyang222**
fixed #440

### CODE REVIEW

File: plugins/huaweicloud/src/main/java/org/eclipse/xpanse/orchestrator/plugin/huaweicloud/monitor/HuaweiCloudMetricsService.java [hunk 1]
```diff
@@ -88,7 +90,15 @@ public List<Metric> getMetricsByResource(ResourceMetricRequest resourceMetricReq
                     huaweiCloudToXpanseDataModelConverter.buildShowMetricDataRequest(
                             resourceMetricRequest, metricInfoList);
             ShowMetricDataResponse showMetricDataResponse =
-                    client.showMetricData(showMetricDataRequest);
+                    client.showMetricDataInvoker(showMetricDataRequest)
+                            .retryTimes(5)
```
Comment (swaroopar) [inline plugins/huaweicloud/src/main/java/org/eclipse/xpanse/orchestrator/plugin/huaweicloud/monitor/HuaweiCloudMetricsService.java:94]: can we move the number of retries to a static constant variable please?

File: plugins/huaweicloud/src/main/java/org/eclipse/xpanse/orchestrator/plugin/huaweicloud/monitor/HuaweiCloudMetricsService.java [hunk 2]
```diff
@@ -88,7 +90,15 @@ public List<Metric> getMetricsByResource(ResourceMetricRequest resourceMetricReq
...
+                            .retryCondition((resp, ex) -> Objects.nonNull(ex)
+                                    && ServiceResponseException.class.isAssignableFrom(
+                                    ex.getClass())
+                                    && (((ServiceResponseException) ex).getHttpStatusCode() == 429
+                                    || ((ServiceResponseException) ex).getHttpStatusCode() == 500))
+                            .backoffStrategy(new HuaweiCloudRetryStrategy(60000))
```
Comment (swaroopar) [inline plugins/huaweicloud/src/main/java/org/eclipse/xpanse/orchestrator/plugin/huaweicloud/monitor/HuaweiCloudMetricsService.java:100]: same for the 60000 value.

**Review by swaroopar — CHANGES_REQUESTED** [review]
comments added.

File: plugins/huaweicloud/src/main/java/org/eclipse/xpanse/orchestrator/plugin/huaweicloud/monitor/HuaweiCloudMetricsService.java (hunk 1 above)
Comment (jinyangyang222) [reply plugins/huaweicloud/src/main/java/org/eclipse/xpanse/orchestrator/plugin/huaweicloud/monitor/HuaweiCloudMetricsService.java]: ok,I made this change.

File: plugins/huaweicloud/src/main/java/org/eclipse/xpanse/orchestrator/plugin/huaweicloud/monitor/HuaweiCloudMetricsService.java (hunk 2 above)
Comment (jinyangyang222) [reply plugins/huaweicloud/src/main/java/org/eclipse/xpanse/orchestrator/plugin/huaweicloud/monitor/HuaweiCloudMetricsService.java]: same here.

**Review by swaroopar — APPROVED** [review]
looks good.

**Pull request merged**